### ISSUE TITLE
Lens still show the Action of Parent

### ISSUE THREAD

**Issue by danielschweiger** [laravel/nova-issues#2902]
- Laravel Version: 8.6
- Nova Version: 3.10
- PHP Version: 7.3
- Database Driver & Version: MySQL 

My lenses Action Function:

```
public function actions(Request $request)
    {
        return [];
    }
```

My lenses parent Action Function:

```
public function actions(Request $request)
    {
        return [
            (new Actions\BookBankTransaction)
                ->confirmButtonText('Next')
                ->cancelButtonText("Cancle")
                ->showOnTableRow()
        ];

    }
```

The Dropdown for running the Action against multiple rows is not available.

But, on the lens view, the Button (showOnTableRow) is still there:

<img width="966" alt="Bildschirmfoto 2020-09-27 um 12 21 21" src="https://user-images.githubusercontent.com/2277612/94362516-1882a980-00bc-11eb-938a-d018c2e56a4a.png">

To reproduce just return 

return [];

in your actions() function in your lens. The parent should have an Action with "showOnTableRow".

**Comment by crynobone** [comment]
Hi @danielschweiger 

Just to confirm, when you refer "parent", is it instance of `Lens` or `Resource`. If the "parent" is `Resource`, laravel/nova#1006 should solved the issue.

**Comment by danielschweiger** [comment]
Hi @crynobone 

The Parent is a Resource.

**Comment by crynobone** [comment]
Fixed in `v3.12.1`
